inst/odin/carehomes.R

```diff
@@ -42,7 +42,9 @@ n_I_P_vaccinated[, ] <-
   sum(n_II_P_next_vacc_class[i, , , j])
```

Comment:
could you add some comments at the start of the odin code to explain how strains 3 and 4 are used i.e. which corresponds to which?